src/main/resources/db/changelog/changes/03-create-roles-table-test-users.yaml

```diff
@@ -0,0 +1,159 @@
+                  value: 1234567891012
+              - column:
+                  name: price
+                  value: 19.12
+              - column:
+                  name: description
+                  value: Harry Potter series, about a young sorcerer in training
+        - insert:
+            tableName: books
+            columns:
+              - column:
+                  name: title
+                  value: Daughter of the Pirate King
+              - column:
+                  name: author
+                  value: Tricia Levenseller
+              - column:
+                  name: isbn
+                  value: 1234567891013
+              - column:
+                  name: price
+                  value: 16.12
+
+        - insert:
+            tableName: users
+            columns:
+              - column:
+                  name: email
+                  value: admin@god.com
+              - column:
+                  name: password
+                  value: $2a$10$T49iZ1kORtzfHBQDCHrvZul6JiWfkbxVKZK31cbS7p7ZVn1d0W5Vm
+              - column:
+                  name: first_name
+                  value: Joe
+              - column:
+                  name: last_name
+                  value: Black
+              - column:
+                  name: shipping_address
+                  value: heaven avenue
+        - insert:
+            tableName: users
+            columns:
+              - column:
+                  name: email
+                  value: user@devil.com
+              - column:
+                  name: password
+                  value: $2a$10$T49iZ1kORtzfHBQDCHrvZul6JiWfkbxVKZK31cbS7p7ZVn1d0W5Vm
+              - column:
+                  name: first_name
+                  value: Keanu
+              - column:
+                  name: last_name
+                  value: Reeves
+              - column:
+                  name: shipping_address
+                  value: New York
+
+        - createTable:
+            tableName: users_roles
+            columns:
+              - column:
+                  name: user_id
+                  type: bigint
+                  constraints:
+                    nullable: false
+              - column:
+                  name: role_id
+                  type: bigint
+                  defaultValue: 2
+                  constraints:
+                    nullable: false
+        - addPrimaryKey:
+            tableName: users_roles
+            columnNames: user_id, role_id
+
+        - insert:
+            tableName: users_roles
+            columns:
+              - column:
+                  name: user_id
+                  value: 1
+              - column:
+                  name: role_id
+                  value: 1
+        - insert:
+            tableName: users_roles
+            columns:
+              - column:
+                  name: user_id
+                  value: 2
+              - column:
+                  name: role_id
+                  value: 2
```

Comment:
I don't think it's good practice to do multiple database operations in a single file

Reply:
one branch - one changelog 
I think another way, this is not a big file for split it up.

Reply:
I agree with @ValentynSharshon , please create anothe changelog, please also use `preConditions:` with `valueComputed:`

Reply:
I'm sorry, but how many changes should there be? Because there we have like be an additional 3, 4 or more, I don’t think that would be better in this situation.